src/test/kotlin/no/nav/familie/ef/iverksett/økonomi/IverksettMotOppdragIntegrasjonsTest.kt

```diff
@@ -0,0 +1,110 @@
+package no.nav.familie.ef.iverksett.økonomi
+
+import io.mockk.every
+import no.nav.familie.ef.iverksett.ServerTest
+import no.nav.familie.ef.iverksett.iverksetting.IverksettingService
+import no.nav.familie.ef.iverksett.iverksetting.tilstand.TilstandRepository
+import no.nav.familie.ef.iverksett.util.opprettBrev
+import no.nav.familie.ef.iverksett.util.opprettIverksett
+import no.nav.familie.kontrakter.ef.iverksett.Periodetype
+import no.nav.familie.prosessering.domene.TaskRepository
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import java.time.LocalDate
+import java.util.UUID
+
+class IverksettMotOppdragIntegrasjonsTest : ServerTest() {
+
+    @Autowired
+    lateinit var oppdragClient: OppdragClient
+
+    @Autowired
+    lateinit var tilstandRepository: TilstandRepository
+
+    @Autowired
+    lateinit var taskRepository: TaskRepository
+
+    @Autowired
+    lateinit var iverksettingService: IverksettingService
+
+    @Autowired
+    lateinit var iverksettMotOppdragTask: IverksettMotOppdragTask
+
+    val behandlingid = UUID.randomUUID()
+    val forsteAndel = lagAndelTilkjentYtelse(
+            beløp = 1000,
+            fraOgMed = LocalDate.of(2021, 1, 1),
+            tilOgMed = LocalDate.of(2021, 1, 31),
+            periodetype = Periodetype.MÅNED)
+    val iverksett = opprettIverksett(behandlingid, andeler = listOf(forsteAndel))
+
+    @BeforeEach
+    internal fun setUp() {
+        every { oppdragClient.iverksettOppdrag(any()) } returns ""
```

Comment:
Trenger vi denne? Isåfall kan vi sjekke "slot" og verifisere den - om det er interessant

Reply:
Sant, jeg vet ikke om vi trenger verifisering av det vi sender til økonomi her. Kanskje en enkel test, men ikke på hver test?

Reply:
Vi sjekker "slot" i IverksettMotOppdragTaskTest. Tror ikke vi trenger å gjøre det her...